contrib/cirrus/cirrus_yaml_test.py

```diff
@@ -26,8 +26,10 @@ def setUp(self):
 class TestDependsOn(TestCaseBase):
 
     ALL_TASK_NAMES = None
-    SUCCESS_DEPS_EXCLUDE = set(['success', 'artifacts', 'podman_machine',
-        'test_image_build', 'release', 'release_test'])
+    SUCCESS_DEPS_EXCLUDE = set(['success', 'artifacts', 'artifacts_aarch64',
+        'podman_machine', 'podman_machine_aarch64',
+        'test_image_build', 'test_image_build_aarch64',
+        'release', 'release_test', 'release_test_aarch64'])
```

Comment:
This isn't right.  I think the `test_image_build_aarch64` maybe shouldn't be here.  i.e. that task should be required for success.

The other additions are reasonable - unless/until you remove `artifacts_aarch64` and `release_test_aarch64` from `.cirrus.yml` as I commented.

Reply:
so, test_image_build isn't required for success, but the aarch64 one is?

Reply:
removed release_test_aarch64 and artifacts_aarch64 from .cirrus.yml